views/apartments/show.erb

```diff
@@ -1 +1,24 @@
-This is info about apartment #3.
+<!-- This is info about apartment #3. -->
+<h2>
+  <%= @apartment %>
+</h2>
+  <p>
+    <label>Address</label>
```

Comment:
Generally, you only use `label` to refer to `input` elements -- you don't use it by itself.

A definition list might work better here:

```
<dl>
  <dt>Address</dt>
  <dd><%= @apartment.address %></dd>
...
</dl>
```